Makefile

```diff
@@ -558,8 +566,25 @@ check-symbols: startup_files libc
 	# This ignores whitespace because on Windows the output has CRLF line endings.
 	diff -wur "$(CURDIR)/expected/$(MULTIARCH_TRIPLE)" "$(SYSROOT_SHARE)"
 
+NORMAL_SYSROOT := \
+	$(findstring $(SYSROOT)/lib,$(SYSROOT_LIB)) \
+	$(findstring $(SYSROOT)/include,$(SYSROOT_INC)) \
+	$(findstring $(SYSROOT)/share,$(SYSROOT_SHARE))
+
+# If the 3 dirs are not within the standard locations, the install rule
+# is undefined. This protects the user from doing something silly by
+# mistake.
+ifeq ($(SYSROOT)/lib $(SYSROOT)/include $(SYSROOT)/share,$(strip $(NORMAL_SYSROOT)))
 install: finish
 	mkdir -p "$(INSTALL_DIR)"
 	cp -r "$(SYSROOT)/lib" "$(SYSROOT)/share" "$(SYSROOT)/include" "$(INSTALL_DIR)"
+.PHONY: install
+endif
```

Comment:
I made `make install` go away for when the directories are in a custom layout, to get rid of the footgun of a broken `make install` doing something the user did not intend.

But per what I wrote in the main thread, and what you wrote in #270, we could just delete `make install` too!